### PR TITLE
[TR] Add `Ranking`

### PR DESCRIPTION
**Pull request by SStoney6378**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

**Comment by ubinici** [comment]
Please refrain from translating stub articles in the future. There are many proper ones that are either outdated or unavailable. It would be a better practice to handle the outdated ones before delving into the other ones.

I'll drop a review at a later time, please do not merge/close the PR by then.

**Comment by TicClick** [comment]
> Please refrain from translating stub articles in the future

while it's generally better to translate completed articles first, we have A LOT of stubs. realistically speaking, it's going to be a long time until all of them are fleshed out, so maybe some of stub translations are worth working on (I would advise against translating anything with `needs_cleanup: true` though)

File: wiki/Ranking/tr.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+stub: true
+etiketler:
```
Comment (WindowsMeosu) [inline wiki/Ranking/tr.md:3]:
```suggestion
tags:
```

**Review by WindowsMeosu — CHANGES_REQUESTED** [review]

**Review by TicClick — APPROVED** [review]
merging (no native review for one week or longer)

**Pull request merged**